api/pkg/resources/address/address_test.go

```diff
@@ -22,7 +22,7 @@ const (
 	fakeClusterNamespaceName = "cluster-ns"
 	// This must be the A record for *.europe-west3c.dev.kubermatic.io and
 	// *.alias-europe-west3-c.dev.kubermatic.io
-	externalIP = "35.198.93.90"
```

Comment:
I'm cherry picking the proper fix for this here: https://github.com/kubermatic/kubermatic/pull/6020

Reply:
Sounds good. I am going to keep it though until that PR is merged, otherwise I won't pass tests.

Reply:
sure